colrev/ops/pages/synthesizedrecords.py

```diff
@@ -0,0 +1,259 @@
+#! /usr/bin/env python3
+"""dashboard table and graphs for synthesized records"""
+from __future__ import annotations
+
+import bibtexparser
+import dash
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from dash import callback
+from dash import dash_table
+from dash import dcc
+from dash import html
+from dash import Input
+from dash import Output
+
+dash.register_page(__name__)
+
+
+with open(
+    "./data/records.bib"
+) as bibtex_file:  # changing file format to csv for pandas
+    bib_database = bibtexparser.load(bibtex_file)
+    if not bib_database.entries:  # checking if bib_database file is empty
+        raise Exception("Die Datei 'records.bib' ist leer.")  # throwing Exception
+df = pd.DataFrame(bib_database.entries)
+df.to_csv("./data/records.csv", index=True)
+data = pd.read_csv("./data/records.csv").query("colrev_status == 'rev_synthesized'")
+data.rename(columns={"Unnamed: 0": "index"}, inplace=True)
+
+for title in data:
```

Comment:
There are short versions to drop all columns except for selected ones from a df ([1](https://www.statology.org/pandas-drop-all-columns-except/)).